lib/KalturaFTPServer.js

```diff
@@ -276,109 +313,164 @@ FtpSession.prototype = {
 	},
 
 	createReadStream: function () {
+		var err = 'createReadStream: Unimplemented usage';
```

Comment:
Please rephrase the error message, no one should know what is implemented and what isn't, the error message should be "Not Authorized" or something.